tables/Tables/TableSyncData.cc

```diff
@@ -36,9 +36,10 @@ TableSyncData::TableSyncData()
 : itsNrrow              (0),
   itsNrcolumn           (-1),
   itsModifyCounter      (0),
-  itsTableChangeCounter (0)
+  itsTableChangeCounter (0),
+  itsMemIO              (new MemoryIO)
```

Comment:
Sometimes I see `new MemoryIO` without parentheses, and sometimes I see `new MemoryIO()` with parentheses. If it works, both are fine. It just suprises me.